src/wasm-builder.h

```diff
@@ -183,52 +182,47 @@ class Builder {
     ret->finalize();
     return ret;
   }
-  Block* makeBlock(const std::vector<Expression*>& items) {
+
+  template<typename T>
+  using bool_if_not_expr_t =
+    std::enable_if_t<std::negation_v<std::is_convertible<T, Expression*>>,
```

Comment:
Btw, any idea why I couldn't get this to work with `std::is_base_of` instead of `is_convertible`?

Reply:
I think it has to do with having to remove the pointer, presumably using `std::remove_pointer`, although TBH I couldn't get that working either, so I'm definitely missing something else there, too.